sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -14,22 +15,50 @@ import life.qbic.portal.sampletracking.resource.ResourceService
 */
 class ProjectOverviewViewModel {
 
-    ObservableList projects = new ObservableList(new ArrayList<Project>())
+    ObservableList projectOverviews = new ObservableList(new ArrayList<ProjectSummary>())
     private final ResourceService<Project> projectResourceService
+    private final ResourceService<StatusCount> statusCountService
 
-    ProjectOverviewViewModel(ResourceService<Project> projectResourceService){
+    ProjectOverviewViewModel(ResourceService<Project> projectResourceService, ResourceService<StatusCount> statusCountService){
         this.projectResourceService = projectResourceService
+        this.statusCountService = statusCountService
         fetchProjectData()
         subscribeToResources()
     }
 
     private void fetchProjectData() {
-        projects.clear()
-        projects.addAll(projectResourceService.iterator())
+        projectOverviews.clear()
+        for (Project project : projectResourceService.iterator()) {
+            addProject(project)
```

Comment:
since when is for cool again? :D why not .each ?

Reply:
For never was uncool.